expression/builtin_time_vec.go

```diff
@@ -772,11 +772,30 @@ func (b *builtinDateDiffSig) vecEvalInt(input *chunk.Chunk, result *chunk.Column
 }
 
 func (b *builtinCurrentDateSig) vectorized() bool {
-	return false
+	return true
 }
 
 func (b *builtinCurrentDateSig) vecEvalTime(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+
+	tz := b.ctx.GetSessionVars().Location()
+	nowTs, err := getStmtTimestamp(b.ctx)
+	if err != nil {
+		return err
+	}
+
+	year, month, day := nowTs.In(tz).Date()
+	mysqlTm := types.FromDate(year, int(month), day, 0, 0, 0, 0)
+
+	n := input.NumRows()
+	result.ResizeTime(n, true)
```

Comment:
We should not set `isNull` to `false` here instead of `true`.

@qw4990 I wonder why the test case can pass even if `isNull` is true?

Reply:
After investigation, I find that our vectorized testing framework can't support functions which has no input argument.
If some function has no input argument, the framework will generate an empty chunk as input data to test, and in this case, this function is not invoked actually.

I will fix this problem soon.

Reply:
@ekalinin I mark this PR as `DNM`(DO NOT MERGE) . We may wait for @qw4990 's patch.

Reply:
Here is the PR to fix this problem [PR/12620](https://github.com/pingcap/tidb/pull/12620).

Reply:
@XuHuaiyu PR/12620 was merged. `DNM` may be removed.

Reply:
`DNM` is removed, but the comment is not resolved.